tracking/traffic_info.cpp

```diff
@@ -0,0 +1,104 @@
+#include "tracking/traffic_info.hpp"
+
+#include "platform/http_client.hpp"
+
+#include "coding/bit_streams.hpp"
+#include "coding/reader.hpp"
+#include "coding/writer.hpp"
+
+#include "base/assert.hpp"
+
+#include "std/algorithm.hpp"
+#include "std/string.hpp"
+
+#include "private.h"
+
+namespace tracking
+{
+namespace
+{
+string const kTestFileName = "traffic_data";
+
+bool ReadRemoteFileAsString(string const & url, string & result)
+{
+  platform::HttpClient request(url);
+  if (request.RunHttpRequest() && request.ErrorCode() == 200)
+  {
+    result = request.ServerResponse();
+    return true;
+  }
+  return false;
+}
+}  // namespace
+
+TrafficInfo::TrafficInfo() {}
```

Comment:
So, I took a look at TrafficInfo_RemoteFile test, and didn't see anything interesting. Could you please explain your last comment?